satpy/tests/reader_tests/modis_tests/test_modis_l3.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Unit tests for MODIS L3 HDF reader."""
+
+from __future__ import annotations
+
+import dask.array as da
+import numpy as np
+import pytest
+from pyresample import geometry
+from pytest_lazyfixture import lazy_fixture
+
+from satpy import Scene, available_readers
+
+from ._modis_fixtures import _shape_for_resolution
+
+
+def _expected_area():
+    proj_param = "EPSG:4326"
+
+    return geometry.AreaDefinition("gridded_modis",
+                                   "A gridded L3 MODIS area",
+                                   "longlat",
+                                   proj_param,
+                                   7200,
+                                   3600,
+                                   (-180, -90, 180, 90))
+
+
+class TestModisL3:
+    """Test MODIS L3 reader."""
+
+    def test_available_reader(self):
+        """Test that MODIS L3 reader is available."""
+        assert "modis_l3" in available_readers()
+
+    @pytest.mark.parametrize(
+        ("loadable", "filename"),
+        [
+            ("Coarse_Resolution_Surface_Reflectance_Band_2", lazy_fixture("modis_l3_nasa_mod09_file")),
+            ("BRDF_Albedo_Parameter1_Band2",lazy_fixture("modis_l3_nasa_mod43_file")),
```

Comment:
@mraspaud why isn't ruff complaining about the lack of a space after the comma?

Reply:
Space is there now :)